tests/integration/test_trace_stats.py

```diff
@@ -20,22 +20,10 @@
 
 
 @pytest.fixture
-def sample_rate():
-    # type: () -> Generator[float, None, None]
-    # Default the sample rate to 0 so no traces are sent for requests.
-    yield 0.0
-
-
-@pytest.fixture
-def stats_tracer(sample_rate):
+def stats_tracer():
     # type: (float) -> Generator[Tracer, None, None]
     with override_global_config(dict(_trace_compute_stats=True)):
         tracer = Tracer()
-        tracer.configure(
-            sampler=DatadogSampler(
-                default_sample_rate=sample_rate,
```

Comment:
Sampling rates other than 0 and 1 don't really work with snapshot tests as far as I know, because they cause the number of sampled spans to vary from test to test.